integration/spark-common-cluster-test/src/test/scala/org/apache/spark/sql/common/util/QueryTest.scala

```diff
@@ -220,4 +218,74 @@ object QueryTest {
 
     return None
   }
+
+  /**
+   * execute the query by establishing the jdbc connection
+   *
+   * @param query
+   * @return
+   */
+  def executeQuery(dbName: String, query: String): List[Map[String, Any]] = {
```

Comment:
Better create new method name or new class for presto

Reply:
created a subclass now.